DropwizardHelloWorld/src/main/java/com/example/helloworld/Saying.java

```diff
@@ -0,0 +1,24 @@
+package com.example.helloworld;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import lombok.AllArgsConstructor;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.hibernate.validator.constraints.Length;
+
+/**
+ * Created by ccardona on 4/11/17.
+ */
+@NoArgsConstructor
+@AllArgsConstructor
+@Getter
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
+@JsonInclude
+public class Saying {
```

Comment:
Really love to see that your annotation kung-fu is getting sharpen. Nice Lombok, Jackson and Validation annotating... Kudos for you 👏